src/index.js

```diff
@@ -189,6 +188,8 @@ class Libp2p extends EventEmitter {
    */
   async start () {
     log('libp2p is starting')
+    // TODO: consider validate listen addresses on start?
+    // depend on transports?
```

Comment:
What do you mean? When the TransportManager listens it results in each Transport filtering out all addresses that don't match it. We're not logging any information around "hey, you gave us an address and there is no transport that's valid for it", unless non of the addresses match any transport.

Reply:
This is not valid anymore. Old thought